### PR TITLE
Feature/Story with Autoscroll

### PR DESCRIPTION
**Pull request by rikaciminieri**
# Description

- What work was done?

An ANTD Card Component was added to the story that the users will read. A scrollable div was implemented with a fixed height. A teal border that matches the color scheme was added. 

- Why was this work done?

This makes the UI cleaner as well as more interactive for the user. Scrolling keeps the reader engaged. The story may eventually present itself with pagination in the future. 

- What feature / user story is it for?

https://trello.com/c/4liJQTnt/818-as-a-reader-i-want-a-more-interactive-reading-experience-so-that-i-can-stay-engaged-with-the-story

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Change Status

- [X] Complete, tested, ready to review and merge

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

##Loom Video
https://www.loom.com/share/cee358d067044fc2a7c1b38d916b9fb6

### CODE REVIEW

File: src/components/pages/GamePlay/GameReadStep.js [hunk 1]
```diff
@@ -41,12 +42,15 @@ export default function GameReadStep(props) {
     <div id="read-step">
       <div id="story" className="gameplay-content">
         <div className="inner-container">
-          <div className="story-title">
-            <h2>Zoom & Boom @ A Saltwater Startup</h2>
-            <span>by Conner Burnham</span>
-          </div>
-
-          <div className="story-content">
+          <Card
```
Comment (KyleCardwell) [inline src/components/pages/GamePlay/GameReadStep.js:45]: Nice use of the antd components. I like how you reorganized the original layout.

**Review by KyleCardwell — APPROVED** [review]
This looks good Rika.  The layout looks and feels similar to the other design styles of the site.  And the functionality of the scroll works well.

**Review by Kishkuman13 — APPROVED** [review]
Rika, this is very clean and simple code. Great aesthetic to match the current style as well. Great work!

**Review by ashtilawat — APPROVED** [review]
Great atomic PR!

**Pull request merged**